shared/redux/rtk-apis/classrooms/classrooms.api.ts

```diff
@@ -35,19 +35,22 @@ const classroomsApi = projectApi.injectEndpoints({
       invalidatesTags: ["Classrooms"],
     }),
 
-    enrollStudent: builder.mutation<TEnrollMentInfo, TEnrollInfo>({
-      query: (enrollmentInfo) => ({
-        url: "classrooms/students",
+    enrollStudent: builder.mutation<TEnrollMentInfo, { id: number; enrollmentInfo: TEnrollInfo }>({
+      query: ({ id, enrollmentInfo }) => ({
```

Comment:
[NIT] - You can keep the type as it was before and do something like this

```suggestion
      query: ({ classroomId, ...rest }) => ({
```